sensor_msgs/PointCloud2 input
---
sensor_msgs/PointCloud2 output
